crates/sui-json-rpc/src/api.rs

```diff
@@ -422,3 +423,23 @@ pub trait EventReadApi {
         end_time: u64,
     ) -> RpcResult<Vec<SuiEventEnvelope>>;
 }
+
+#[open_rpc(namespace = "sui", tag = "Quorum Driver APIs to execute transactions.")]
+#[rpc(server, client, namespace = "sui")]
+pub trait QuorumDriverApi {
+    /// Execute the transaction and wait for results if desired
+    #[method(name = "executeTransaction")]
+    async fn execute_transaction(
+        &self,
+        /// transaction data bytes, as base-64 encoded string
+        tx_bytes: Base64,
+        /// Flag of the signature scheme that is used.
+        sig_scheme: SignatureScheme,
+        /// transaction signature, as base-64 encoded string
+        signature: Base64,
```

Comment:
not sure if you've rebased recently, but be aware that signature is now part of the signed data, so this should perhaps be refactored to reflect that.

Reply:
initially I tried to use `TransactionData` v.s. this bunch of things but it did not work out for some doc reasons. If I recall correctly, something in move-lang repo is not compatible with how we generate our api docs and refuse to compile .. @patrickkuo  do you have some ideas whether/how this can be fixed?

Reply:
The types in message.rs are not json/schema friendly, we have created another set of rpc types mirroring them, e.g. TransactionData <> SuiTransactionData, we can create a `SuiSenderSignedData` for this.